app/views/declaration/items_summary.scala.html

```diff
@@ -30,79 +42,142 @@
-                    <td id="item_@index--item_type" >@item.commodityDetails.map(_.combinedNomenclatureCode) </td>
-                    <td id="item_@index--package_count">
-                        <ol type="1">@item.packageInformation.getOrElse(List.empty).map(packageInfo =>
-                            packageInfo.numberOfPackages.map(packages => <li>{packages}</li>)
-                        )
-                        </ol>
-                    </td>
-                    <td>
-                        <a id="item_@index--change" href="@routes.ProcedureCodesController.displayPage(mode, item.id)">
-                            <span aria-hidden="true">@messages("site.change")</span>
-                            <span class="visuallyhidden">@{messages("supplementary.itemsAdd.change.hint", item.sequenceId)}</span>
-                        </a>
-                    </td>
-                    <td>
-                        <a id="item_@index--remove" class="button button--gray" href="@routes.ItemsSummaryController.removeItem(mode, item.id)">
-                            <span aria-hidden="true">@messages("site.remove")</span>
-                            <span class="visuallyhidden">@{messages("supplementary.itemsAdd.remove.hint", item.sequenceId)}</span>
-                        </a>
-                    </td>
-                </tr>
-            }
-
-            </tbody>
-        </table>
+        @saveAndContinue("site.save_and_continue")
     }
-    @helper.form(routes.ItemsSummaryController.submit(mode), 'autocomplete -> "off") {
-        @helper.CSRF.formField
-
-        <div class="section">
-            @if(items.isEmpty) {
-                @components.buttons.add_button("site.add.item", "site.add.item")
-            } else {
-                @components.buttons.add_button("site.add.anotherItem", "site.add.anotherItem")
-            }
+    <p class="govuk-body">
+        @saveAsDraft("site.save_and_come_back_later")
+    </p>
+}
+
+@changeLink(item: ExportItem) = {
+    <a class="govuk-link" href="@routes.ProcedureCodesController.displayPage(mode, item.id).url">
+        @messages("site.change")
+        <span class="govuk-visually-hidden">
+            @messages("supplementary.itemsAdd.change.hint", item.sequenceId)
+        </span>
+    </a>
+}
 
-            @if(items.nonEmpty) {
-                @components.buttons.save_and_continue_button()
+@removeButton(item: ExportItem) = @{
+    govukButton(Button(
+        classes = "govuk-button--secondary govuk-!-margin-0",
+        content = Text(messages("site.remove")),
+        name = Some(messages("supplementary.itemsAdd.remove.hint", item.sequenceId)),
+        href = Some(routes.ItemsSummaryController.removeItem(mode, item.id).url)
+    ))
+}
+
+@noOfPackagesSection(item: ExportItem) = {
+    @item.packageInformation.map { packageInformationList =>
+        @if(packageInformationList.size == 1) {
```

Comment:
Here you need to have `< 2` or `.drop(1).isEmpty)`

Actually you didn't cover the situation when list is empty.
If it's empty you will go to else and render `<ol>`

Reply:
Strangely, that code has it covered when the `packageInformationList` is empty.
:shrug:

Reply:
I have addressed your concern

Reply:
I think the code will work, but when you check the html source you'll have the empty `<ol>